OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -304,6 +304,12 @@ public bool CanEnterCell(World world, Actor self, CPos cell, out int movementCos
 		bool IOccupySpaceInfo.SharesCell { get { return SharesCell; } }
 	}
 
+	public interface INotifyMovement
```

Comment:
Please move this to `TraitsInterfaces.cs`, as this file is horrible as it is, with two monsterous classes, an enum and nested classes :X